torch_em/transform/augmentation.py

```diff
@@ -1,6 +1,11 @@
-import torch
+from typing import List, Optional, Sequence, Tuple, Union
+
+import kornia.augmentation.utils
 import numpy as np
-import kornia
```

Comment:
Can we leave the single import please instead of splitting it into 5 different ones?